app/app.rb

```diff
@@ -21,6 +21,9 @@
 require_relative "tint_omniauth" # Monkeypatch
 require_relative "helpers"
 
+ENV["GIT_COMMITTER_NAME"] = "Tint"
```

Comment:
The gem just exec's git, and these environment variables are the way git chooses a committer.